store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java

```diff
@@ -36,9 +33,9 @@
 
     private boolean multiCommitLogPathEnable = false;
 
-    private List<String> commitLogStorePaths = null;
+    private String commitLogStorePaths = null;
```

Comment:
Why change List<String> to Sting ?  Repeatedly parse this parameter every u need use it.